lib/modules/manager/flux/types.ts

```diff
@@ -1,8 +1,8 @@
 import type { FluxResource } from './schema';
 
-export type FluxManagerData = {
-  components: string;
-};
+export interface FluxManagerData {
+  components?: string;
```

Comment:
Why is this suddenly optional?

Reply:
otherwise we'll get typescript failures:
![image](https://github.com/user-attachments/assets/32bbf923-91fd-49d6-acb1-8431d3935a50)